todo-reactjs/src/components/TodoApp.js

```diff
@@ -13,11 +13,12 @@ export default class TodoApp extends Component {
 	}
 
 	handleChange = e => {
-		this.setState({
-			[e.target.name]: e.target.value
-		});
+		this.setState({[e.target.name]: e.target.value})
 	}
 
+	onEnterKeyPress = e => {
+		e.key === 'Enter' ? this.addTodoToList(this.state.todoItem) : null
```

Comment:
@prakashsejwani 

Is there anyways better than this to write this expression?
Just because I don't like that `null` thing at the end.